.circleci/config.yml

```diff
@@ -78,6 +78,23 @@ jobs:
       - run: { name: 'Release S3', command: "cd patternlab/styleguide && node_modules/.bin/gulp patternlab:release" }
       - run: { name: 'Release NPM', command: "cd patternlab/styleguide && node_modules/.bin/gulp npm:release" }
 
+  react_build:
+    working_directory: /var/www/code
+    docker:
+      - image: circleci/node:8.9.4-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - mf-react-dependencies-{{ checksum "package.json" }}
+      - run: { name: 'NPM Install', command: 'cd react && npm install' }
+      - save_cache:
+          name: Save NPM cache
+          key: mf-react-dependencies-{{ checksum "package.json" }}
+          paths:
+            - node_modules
+      - run: npm run storybook
```

Comment:
`cd react && npm run storybook`

Reply:
Also, the storybook script starts a server, right?  I think that'll be a problem here.  Would `build-storybook` work?

Reply:
Yea, I think this will need to be `build-storybook` & that build is what the nginx server will expose for the test job?

Reply:
Yeah, exactly.  You want to build to static HTML, then have the nginx server serve it.